src/api/public/apidocs-new/paths/missing_owner.yaml

```
@@ -0,0 +1,62 @@
+get:
+  summary: |
+    Search for packages that miss the definition for a by default
+    responsible person or group.
+  description: |
+    Search for packages that miss the definition for a by default
+    responsible person or group.
```

```suggestion
  summary: |
    Search for packages that miss the definition for a by default
    responsible person or group.
```
If the description doesn't provide any additional value to the summary, I would remove it.